pkg/common/utils.go

```diff
@@ -71,6 +73,13 @@ func VolumeIDToKey(id string) (*meta.Key, error) {
 	}
 }
 
+func GenerateUnderspecifiedVolumeID(diskName string, isZonal bool) string {
+	if isZonal {
```

Comment:
How can you know that a disk is zonal or regional and not know the zone/region?

We do not support regional disks for inline volumes

Reply:
Ok I see that this is only being used by the e2e test.  Can we move this function there instead?

Reply:
I actually want to use it in the translation library as well. Apparently we can tell whether a disk is zonal or regional based on the PV (which we have access to in the translation library). If it is inline, its always zonal

Reply:
If you're translating from a PV, you can already extract the zones from the label.

Reply:
although its true that I guess in that case we will know the zone/region... is it true that we can always get the zone/zones on the PV for GCE PD? I think it is based on the code I've seen. In that case we may only hit the UNSPECIFIED case for in-line volumes. I still want this util function to distinguish between zonal/regional regardless

Reply:
If the labels were missing, then the PV would be likely non-functional in regional clusters.  Reasons why it may be missing:
* User modified the PV afterwards to remove/edit the labels
* PV was created in an old version that didn't have the admission controller running.

Reply:
Ok cool, either way this PR as a whole makes the driver resilient to unspecified values. In the case where they are specified there is no difference in functionality